staging/src/k8s.io/apiserver/pkg/server/egressselector/config.go

```diff
@@ -103,27 +102,13 @@ func ValidateEgressSelectorConfiguration(config *apiserver.EgressSelectorConfigu
 		}
 	}
 
-	var foundControlPlane, foundMaster bool
 	for _, service := range config.EgressSelections {
```

Comment:
does this error if the same name is specified twice in the list? I think it should with a duplicate error

Reply:
Added.